posthog/models/feature_flag/flag_matching.py

```diff
@@ -281,6 +294,14 @@ def get_matches(self) -> tuple[dict[str, Union[str, bool]], dict[str, dict], dic
                     continue
             try:
                 flag_match = self.get_match(feature_flag)
+
+                flags_details[feature_flag.key] = FeatureFlagDetails(
+                    match=flag_match,
+                    id=feature_flag.id,
+                    version=feature_flag.version or 0,
```

Comment:
do you want un-versioned flags to be 0 or 1?  What's the DB default; it's 1 now, right?

Reply:
Fixed! Nice catch!